core-test/src/test/scala/org/bitcoins/core/script/arithmetic/ArithmeticInterpreterTest.scala

```diff
@@ -514,18 +514,14 @@ class ArithmeticInterpreterTest extends FlatSpec with MustMatchers {
   it must "evaluate an OP_WITHIN correctly" in {
     val stack = List(ScriptNumber(2), ScriptNumber.one, ScriptNumber.zero)
     val script = List(OP_WITHIN)
-    val program = ScriptProgram(
-      ScriptProgram(TestUtil.testProgramExecutionInProgress, stack, script),
-      Seq[ScriptFlag]())
+    val program = ScriptProgram(TestUtil.testProgramExecutionInProgress, stack, script).addFlags(Seq[ScriptFlag]())
```

Comment:
We should use `Seq.empty` rather than `Seq[ScriptFlag]()`, this avoids an allocation of a `Seq` object that is empty.